File: crescer/java/trabalhos/tcc/api/src/main/java/br/com/cwi/crescer/api/controller/request/CreateNewUserRequest.java
package br.com.cwi.crescer.api.controller.request;

import br.com.cwi.crescer.api.model.Permission;
import lombok.Data;

import javax.validation.constraints.NotEmpty;


@Data
public class CreateNewUserRequest {
    @NotEmpty
    private String userName;
    @NotEmpty
    private String nickName;
    @NotEmpty
    private String email;
    @NotEmpty
    private String  password;

    private String profileImg;

    private Permission permission;
}

File: back/healthCare/src/main/kotlin/com/healthCare/api/controller/request/CreateNewUserRequest.kt
package com.healthCare.api.controller.request

import javax.validation.constraints.Email
import javax.validation.constraints.NotEmpty
import javax.validation.constraints.Null

data class CreateNewUserRequest(

    @Null
    val id:Long,

    @Email
    val email:String,

    @NotEmpty
    val password:String

)
